vetomint/src/lib.rs

```diff
@@ -139,13 +135,16 @@ pub struct HeightInfo {
     pub validators: Vec<VotingPower>,
 
     /// The index of this node
-    pub this_node_index: ValidatorIndex,
+    pub this_node_index: Option<ValidatorIndex>,
```

Comment:
Add some descriptive comment here

Reply:
insert "validator index can be none for supporting non-validator client" for additional description